--- scons/libs/clam_build_helpers.py.lib64	2010-02-09 21:32:28.000000000 +0100
+++ scons/libs/clam_build_helpers.py	2011-04-10 12:01:53.000000000 +0200
@@ -1,4 +1,4 @@
-import sys
+import sys, sysconfig
 
 #---------------------------------------------------------------
 # from file_retriever.py
@@ -110,8 +110,8 @@
 		if environ['prefix_for_packaging'] != '.':
 			self.prefix = environ['prefix_for_packaging'] 
 		else: self.prefix = environ['prefix']
-		self.lib  = self.prefix + '/lib'
-		self.pc   = self.prefix + '/lib/pkgconfig'
+		self.lib  = self.prefix + '/' + sysconfig.get_config_var('LIB')
+		self.pc   = self.prefix + '/' + sysconfig.get_config_var('LIB') + '/pkgconfig'
 		self.bin  = self.prefix + '/bin'
 		self.inc  = self.prefix + '/include'
 		self.data = self.prefix + '/share'
--- scons/sconstools/pc.py.lib64	2010-01-21 09:43:18.000000000 +0100
+++ scons/sconstools/pc.py	2011-04-10 12:05:17.000000000 +0200
@@ -1,9 +1,9 @@
-import os
+import os, sysconfig
 
 pkgConfigTemplate = """\
 name = @name@
 prefix = @prefix@
-libdir = ${prefix}/lib
+libdir = ${prefix}/@lib@
 includedir = ${prefix}/include
 
 Name: ${name}
@@ -21,6 +21,7 @@
 		TEXTFILESUFFIX='.pc',
 		SUBST_DICT=[
 			('@prefix@', prefix ),
+			('@lib@', sysconfig.get_config_var('LIB')),
 			('@name@', package ),
 			('@version@', version ),
 			('@description@', "C++ Framework for analysis, synthesis and transformation of music audio signals" ),
